### PR TITLE
feature/BDD_GWT_IP2-543_Creation_of_Calendar_Part_1

### PR DESCRIPTION
**Pull request by katkoch**
creation of new scenarios

### CODE REVIEW

File: SeleniumProject/Features/Calendar.feature [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+@ignore
+# Zephyr link
+Scenario: CALENDAR_Part1_2_IP2-543_Every_date_can_be_highlighted
+When User click on randomly picked date
+Then date is highlighted
+
+@ignore
+#Zephyr link
+Scenario: CALENDAR_Part1_3_IP2-544_Months_can_be_changed 
+When User changes <month>
```
Comment (Iga221) [inline SeleniumProject/Features/Calendar.feature:22]: '<month>'

Comment (Iga221) [inline SeleniumProject/Features/Calendar.feature:22]: '<month>'

Comment (Iga221) [inline SeleniumProject/Features/Calendar.feature:22]: '<month>'

Comment (Iga221) [reply SeleniumProject/Features/Calendar.feature:22]: '<month>'

**Review by Iga221 — APPROVED** [review]

File: SeleniumProject/Features/Calendar.feature (hunk 1 above)
Comment (katkoch) [reply SeleniumProject/Features/Calendar.feature]: added

File: SeleniumProject/Features/Calendar.feature [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+#https://tracker.intive.com/jira/browse/IP2-728
+Scenario: CALENDAR_Part1_1_IP2-543_Present_day_is_highlighted
+Then Present day is highlighted
+
+@ignore
+# https://tracker.intive.com/jira/browse/IP2-729
+Scenario: CALENDAR_Part1_2_IP2-543_Every_date_can_be_highlighted
+When User click on randomly picked date
+Then date is highlighted
```
Comment (Iga221) [inline SeleniumProject/Features/Calendar.feature:17]: Maybe better "selected date is highlighted"?

**Review by Iga221 — APPROVED** [review]

**Pull request merged**